transformer_engine/jax/fp8.py

```diff
@@ -48,7 +48,13 @@ def is_fp8_available(gpu_id=None) -> Tuple[bool, str]:
         _is_fp8_available = True
         devices = jax.local_devices()
         for gpu in devices:
-            ret, msg = _check_fp8_support(gpu.id)
+            # WAR a jax issues, the device_id isn't always right!
+            if len(devices) == 1:
+                gid = 0
+            else:
+                gid = gpu.id
+
+            ret, msg = _check_fp8_support(gid)
```

Comment:
It seems `gpu.id` represents a global device index rather than a local index, so it isn't right in the case with multiple GPUs per proc:
```suggestion
        for gpu_id, _ in enumerate(devices):
            ret, msg = _check_fp8_support(gpu_id)
```